src/Modal/PopperElement.jsx

```diff
@@ -30,7 +30,8 @@ PopperElement.defaultProps = {
 PopperElement.propTypes = {
   children: PropTypes.node,
   target: PropTypes.shape({
-    current: PropTypes.node,
+    // eslint-disable-next-line react/forbid-prop-types
+    current: PropTypes.object,
```

Comment:
This is a common issue with eslint 😄, you should use `PropTypes.shape({})` instead to indicate that you expect an object. (and eslint won't complain then)

Reply:
Corrected